sonar-ws/src/main/java/org/sonarqube/ws/client/custommeasures/MetricsRequest.java

```diff
@@ -17,43 +17,50 @@
  * along with this program; if not, write to the Free Software Foundation,
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
-package org.sonarqube.ws.client.qualitygate;
+package org.sonarqube.ws.client.custommeasures;
 
-import javax.annotation.CheckForNull;
-import javax.annotation.Nullable;
+import java.util.List;
+import javax.annotation.Generated;
+
+/**
+ * List all custom metrics for which no custom measure already exists on a given project.<br /> The project id or project key must be provided.<br />Requires 'Administer System' permission or 'Administer' permission on the project.
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Split this 231 characters long line (which is greater than 180 authorized). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS00103)